src/stylesheets/components/_header.scss

```diff
@@ -25,7 +25,7 @@ $z-index-nav:     9000;
   .usa-search {
     @include media($nav-width) {
       float: right;
-      max-width: 21.5rem;
+      max-width: 27em; // max-width of 27 letters
```

Comment:
we use this in two places, and have a comment explaining it, so it seems like a good candidate for a variable!

Reply:
Added in a2c60bb.